### PR TITLE
Improve project explained columns format and Fix compile errors

### PR DESCRIPTION
**Pull request by yixinglu**
as title.

### CODE REVIEW

File: src/planner/Query.cpp [hunk 1]
```diff
@@ -153,7 +154,14 @@ void Project::clone(const Project &p) {
...
     auto desc = SingleInputNode::explain();
-    addDescription("columns", cols_ ? cols_->toString() : "", desc.get());
+    if (cols_) {
+        auto columns = folly::dynamic::array();
+        for (const auto* col : cols_->columns()) {
+            DCHECK(col != nullptr);
+            columns.push_back(col->toString());
+        }
+        addDescription("columns", folly::toJson(columns), desc.get());
+    }
```
Comment (Shylock-Hg) [inline src/planner/Query.cpp:164]: else ?

Comment (yixinglu) [reply src/planner/Query.cpp:164]: do nothing and don't output columns info

**Review by jievince — DISMISSED** [review]

File: src/planner/Query.cpp (hunk 1 above)
Comment (Shylock-Hg) [reply src/planner/Query.cpp:164]: Will empty list is more clear?

Comment (yixinglu) [reply src/planner/Query.cpp:163]: Fixed

**Review by jievince — APPROVED** [review]

**Review by Shylock-Hg — APPROVED** [review]

**Pull request merged**